### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by gallexi**
- [ ] add a check box
body edit

### CODE REVIEW

**Comment by anti-gallexi** [comment]
can I still edit my own

**Comment by gallexi** [comment]
more comments

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# new-repo
\ No newline at end of file
+# new-repo
+
```
Comment (gallexi) [inline README.md:2]: this is a REVEWI COMMENT

**Review by gallexi — COMMENTED** [review]
THIS IS a review edit

**Pull request closed without merging**